_foundations/almalinux.md

```diff
@@ -0,0 +1,48 @@
+---
+identifier: AlmaLinux
+commonName: AlmaLinux
+legalName: AlmaLinux OS Foundation
+website: https://almalinux.org
+foundingDate: '2021'
+location: 'US'
+newProjects: 
+softwareType: distro
+wikidataId: Q106601457
+taxID: 86-2791864
+nonprofitStatus: Nonprofit501c6
+boardSize: 9
+boardType: elected
+boardurl: https://wiki.almalinux.org/Transparency.html
+bylawsurl: https://almalinux.org/p/foundation-bylaws/
+numberOfEmployees: 0
+funding: sponsors
+budgetUsd: 
+budgetYear:
+budgeturl: 
+sponsorurl: https://almalinux.org
+sponsorList: Hawk Host, OpenLogic, CERN, black.host, procomputers.com, Hivelocity, Mattermost, Cybertrust Japan Co., Codenotary, Atix, World4You Internet Services GmbH, CloudLinux Inc, KnownHost, LLC, WebPros, Inc (under cPanel and Plesk brands), IPInfo
+licenses: https://www.osgeo.org/about/licenses/ 
+claPolicy: 
+ethicsPolicy:https://almalinux.org/p/foundation-board-operations-and-ethics/
```

Comment:
Typo, needs a space after the first colon (we still need to add linting to commit checks)